lightwood/api/predictor.py

```diff
@@ -454,6 +448,7 @@ def predict(self, when_data=None, when=None):
                         if 'confidences' in helper_mixer_predictions[output_column] and helper_mixer_predictions[output_column]['confidences'] is not None:
                             main_mixer_predictions[output_column]['confidences'] = list(helper_mixer_predictions[output_column]['confidences'])
 
+        print(main_mixer_predictions)
```

Comment:
prints

Reply:
Will remove, will also remove a bunch of other redundant stuff in the numerical encoder and the quantile loss since now I have a separate branch for predicting numerical ranges.